src/sentry/integrations/source_code_management/repository.py

```diff
@@ -144,7 +183,29 @@ def get_codeowner_file(
             if html_url:
                 try:
                     contents = self.get_client().get_file(repo, filepath, ref, codeowners=True)
+                    metrics.incr(
+                        REPOSITORY_INTEGRATION_GET_FILE_METRIC.format(result="success"),
+                        tags={"integration": self.integration_name},
+                    )
                 except ApiError:
+                    metrics.incr(
```

Comment:
It might also be good to include some additional tag data here, like HTTP status code, just so we have some rough idea of the types of failures we're getting.

Reply:
Though I also understand that cardinality is a concern, so perhaps we could restrict it to a small number of the most important codes.